docs/index.md

```diff
@@ -1,4 +1,4 @@
-<center>![images/logo.svg](images/logo.svg)</center>
+<center><img src="images/logo.svg" alt="logo" width="350"/></center>
```

Comment:
I'm not a frontend developer but specifying dimensions as numerical constants in pixels is always a bad idea,
please use either auto scaling or ratio.